integration-templates/teamtailor/teamtailor-candidates.ts

```diff
@@ -11,7 +11,7 @@ export default async function fetchData(nango: NangoSync): Promise<void> {
                 link_path_in_response_body: 'links.next',
                 limit_name_in_request: 'page[size]',
                 response_path: 'data',
-                limit: 100
+                limit: 30
```

Comment:
is there any linear issue associated with this change? 
if so can you think about linking them in your PR desc, that would be great to add some more context ☺️

Reply:
Thanks @bodinsamuel for approving this 🙏. I will do that with the upcoming pull requests ☺️.